diffrax/_brownian/path.py

```diff
@@ -121,25 +122,26 @@ def _evaluate_leaf(
         t1: RealScalarLike,
         key,
         shape: jax.ShapeDtypeStruct,
-        levy_area: str,
+        levy_area: type[Union[TimeLevyArea, SpaceTimeLevyArea]],
         use_levy: bool,
     ):
         w_std = jnp.sqrt(t1 - t0).astype(shape.dtype)
+        w = jr.normal(key, shape.shape, shape.dtype) * w_std
+        dt = t1 - t0
 
-        if levy_area == "space-time":
+        if issubclass(levy_area, SpaceTimeLevyArea):
```

Comment:
Nit: for clarity, I think I'd probably go for the stricter `if levy_area is SpaceTimeLevyArea` here. We shouldn't be allowing strict subclasses of `SpaceTimeLevyArea` anyway.

Reply:
I agree, the only minor issue is that when doing `levy_area is SpaceTimeLevyArea` pyright can't seem to narrow `levy_area` down to a `SpaceTimeLevyArea`. 
```python
if levy_area is SpaceTimeLevyArea:
    reveal_type(levy_area) # type[TimeLevyArea] | type[SpaceTimeLevyArea]
    levy_area(dt, w, h) # PyRight complains that `h` is in the call signature.
    SpaceTimeLevyArea(dt, w, h) # This explicit call to the initialiser is fine.
elif levy_area is TimeLevyArea:
    reveal_type(levy_area) # type[TimeLevyArea] | type[SpaceTimeLevyArea]
    levy_area(dt, w) # PyRight complains that `h` is missing from the call signature.
```
I presume this is a bug in PyRight, but I'm not sure. In any case, I can replace the `levy_area` with explicit calls to the appropriate initialiser?

Reply:
Ah, I'd forgotten about this. This is an intentional choice in pyright. (Although not one I agree with.)
In this case let's stick with `issubclass` then!